stable/database/templates/statefulset.yaml

```diff
@@ -195,8 +195,10 @@ spec:
         configMap:
           name: {{ template "database.fullname" . }}-configuration
       {{- end }}
+      {{- if not .Values.database.sm.logPersistence.enabled }}
       - name: log-volume
         emptyDir: {}
+      {{- end }}
```

Comment:
does the SM log anything to this dir?

Reply:
log volume is still there but replaced with the persistent volume (still /var/log/nuodb)

Reply:
I can't see logs but the crashdir is inside

Reply:
I believe that engines will store core files inside only.